specification/synapse/resource-manager/readme.md

```diff
@@ -28,18 +28,38 @@ These are the global settings for the Azure Synapse Analytics API.
 description: Azure Synapse Analytics Management Client
 openapi-type: arm
 azure-arm: true
-tag: package-preview-2021-04
+tag: package-2021-05-preview
 generate-empty-classes: true
 modelerfour:
   lenient-model-deduplication: true
 ```
 
+### Tag: package-2021-05-preview
 
-### Tag: package-preview-2021-04
+These settings apply only when `--tag=package-2021-05-preview` is specified on the command line.
 
-These settings apply only when `--tag=package-preview-2021-04` is specified on the command line.
+``` yaml $(tag) == 'package-2021-05-preview'
+input-file:
+  - Microsoft.Synapse/preview/2021-04-01-preview/bigDataPool.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/checkNameAvailability.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/firewallRule.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/integrationRuntime.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/keys.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/library.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/operations.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privateEndpointConnections.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privateLinkResources.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/privatelinkhub.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/sqlPool.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/sqlServer.json
+  - Microsoft.Synapse/preview/2021-04-01-preview/workspace.json
+```
```

Comment:
Is there any particular reason that we have to name this combination of swaggers full of 2021-04-01-preview api-version  `package-2021-**05**-preview`?

And also what is the difference between this tag and `package-2021-04-preview`? These two basically consist of the same swaggers

Reply:
We removed kustopool.json in this *05* tag because that API set is not ready for SDK generation.

Reply:
This does not make sense. If it is not ready, we should remove it in 2021-04-01-preview tag, despite this will introduce breaking changes. On the other hand, tag 2021-04-01-preview is a preview tag, breaking changes should be expected, especially we could regard this as a bug fix.

Reply:
Thanks for the detailed suggestions. I have removed `package-2021-05-preview`